ethereum/ts-scripts/relayer/config/testnet/scriptConfigs/configureDeliveryProvider.json

```diff
@@ -2,53 +2,35 @@
   "addresses": [
     {
       "chainId": 6,
-      "rewardAddress": "0x61a51662f0B30Bf176484cAc5B4a033C497cB2f3",
-      "approvedSenders": [
```

Comment:
what were this `approvedSenders`?

Reply:
In an ancient version of the AR, you could actually designate relayer accounts and only those would be able to relay on behalf of the specific delivery provider. That has since been removed. I'm not sure why it was implemented in the first place since relayer fee payments are finalized in the source chain rather than the destination chain so MEV, front running and oracle copying of relays is not a concern.